webgpu.h

```diff
@@ -1539,6 +1561,14 @@ typedef struct WGPUSurfaceConfiguration {
      * When and in which order the surface's frames will be shown on the screen. Defaults to @ref WGPUPresentMode_Fifo.
      */
     WGPUPresentMode presentMode;
+    /**
+     * Color space to display the surface with.
+     *
+     * The only value that implementations are required to support is @ref WGPUColorSpace_Undefined,
+     * which may be either "unmanaged" or "sRGB".
+     * Wasm implementations should support @ref WGPUColorSpace_SRGB and @ref WGPUColorSpace_DisplayP3.
+     */
+    WGPUColorSpace colorSpace;
```

Comment:
Should we have the surface capabilities reflect the available color spaces?

Reply:
We talked about this in https://github.com/webgpu-native/webgpu-headers/issues/200#issuecomment-1878100315 but it seemed complex in practice and not necessary yet. On the web we currently just support all three formats with all two color spaces, but in native I don't know if all of those combinations are possible, and certainly new color spaces might not be compatible with all formats or vice versa.

Reply:
Approve then? :)
(docs updated)

Reply:
filed #366 for both colorspace and tonemapping capabilities